tests/src/Functional/DeveloperTest.php

```diff
@@ -171,12 +181,37 @@ protected function developerRegisterTest() {
 
     // Try to register with correct API credentials.
     $this->restoreKey();
+
+    $account = $this->entityTypeManager->getStorage('user')->create([
+      'mail' => $test_user['email'],
+      'name' => $test_user['username'],
+      'first_name' => $test_user['first_name'],
+      'last_name' => $test_user['last_name'],
+    ]);
+
+    $this->entityTypeManager->getStorage('user')->resetCache();
+    $this->developerStorage->resetCache();
+
+    // Add matched organization response so it returns the org whenever called.
+    $this->addOrganizationMatchedResponse();
```

Comment:
This test and any other tests that call `addOrganizationMatchedResponse()` fail due to the matcher being serialized which has a closure added to it

```
web % ../vendor/bin/phpunit modules/contrib/apigee_edge/tests/src/Functional/DeveloperTest.php
PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

Testing Drupal\Tests\apigee_edge\Functional\DeveloperTest
E                                                                   1 / 1 (100%)

Time: 9.24 seconds, Memory: 8.00 MB

There was 1 error:

1) Drupal\Tests\apigee_edge\Functional\DeveloperTest::testDeveloperRegisterAndCreate
Exception: Serialization of 'Closure' is not allowed

web/core/lib/Drupal/Component/Serialization/PhpSerialize.php:14
web/core/lib/Drupal/Core/KeyValueStore/DatabaseStorage.php:115
web/core/lib/Drupal/Core/State/State.php:84
web/modules/contrib/apigee_edge/tests/modules/apigee_mock_api_client/src/DatabaseMockStorage.php:180
vendor/apigee/apigee-mock-client-php/src/MockStoragePropertyTrait.php:62
/Users/chrisnovak/projects/drupal8/edge-web/modules/contrib/apigee_edge/tests/modules/apigee_mock_api_client/tests/src/Traits/ApigeeMockApiClientHelperTrait.php:146
/web/modules/contrib/apigee_edge/tests/src/Functional/DeveloperTest.php:196
web/modules/contrib/apigee_edge/tests/src/Functional/DeveloperTest.php:154

ERRORS!
Tests: 1, Assertions: 4, Errors: 1.
```

Reply:
Please test using this PR on the mock client library: https://github.com/apigee/apigee-mock-client-php/pull/3